src/main/kotlin/repository/ProjectTableRepo.kt

```diff
@@ -21,6 +23,11 @@ import snowballr.ProjectOuterClass
  */
 interface IProjectTableRepo {
     suspend fun createProject(request: ProjectOuterClass.Project.Create, userId: String): Project
+
+    /**
+     * Returns all active projects stored on the server
```

Comment:
```suggestion
     * Returns all active projects stored in the database
```
Because we are on the repository layer, imo "in the database" is more accurate than "on the server", what do you think?

Reply:
Sounds better, I agree.